app/api/routes.py

```diff
@@ -251,6 +256,34 @@ def update_votes(id, vote_direction):
     return standardize_response(payload=dict(data=resource.serialize))
 
 
+def add_click(id):
+    try:
+        resource = Resource.query.get(id)
+
+        if not resource:
+            return redirect('/404')
+
+    except MultipleResultsFound as e:
+        print_tb(e.__traceback__)
+        logger.exception(e)
```

Comment:
I've never been able to actually create a `MultipleResultsFound` condition. But let's suppose that it happened, what would this function return?

If this was basically copy-pasted from other parts of the routes file, we should probably open an issue that these error conditions should return early with some sort of error to the client. Maybe a `500` would do since we're logging the error?

Not a blocker for the PR, but something we should think about

Reply:
Ah yes, I did copy and paste this from another route (upvotes).

Yeah I don’t know how it would be possible to find multiple resources with the same id. That would be impossible database-wise. Maybe we should remove this condition altogether?

Reply:
Considering that I've never run into it and I can't produce it for unit tests, it would be good for test coverage purposes to go ahead and just delete it on all routes. I agree that it makes no sense that it should ever return multiple results when we have `id` as the primary key and specified as unique in the model. If this ever happens, I hope it just crashes everything and we are forced to clean up the data lol.

Please open an issue documenting this or just a PR to remove all of them. If we run into some strange issue in prod, I'll take on the task/responsibility of troubleshooting and fixing it. Thanks for the second set of eyes on this.

Reply:
Sounds good! I’ll create an issue for this for documentation sake and prolly submit the PR tomorrow.

Reply:
Awesome, thanks! You've been such a great help to pushing this over the finish line, I very much appreciate that. This is the most momentum we've gotten for the project in months and it couldn't have come at a better time

Reply:
My pleasure! I’m happy to contribute! This is the first time I’ve contributed to open source and it’s been an awesome experience so far :D

Reply:
That makes me feel good. At my day job, I'm on the "Developer Experience" team, maintaining our API and open source client libraries that our customers use to interact with it. Our number 1 mission is to make contributing to and using our code a delightful and seamless experience, and I'm glad that I can bring that same type of delight to contributors to Operation Code because it's an organization I care a lot about. 

Anyway, this project is just about ready to go live and it's my first open source project that I personally came up with and saw from design to production, so this is huge for me. :excited: